glowfic-dl.py

```diff
@@ -207,13 +308,30 @@ async def download_chapter(
     return (title, list(render_posts(posts, image_map, authors)))
 
 
-def compile_chapters(chapters: list[tuple[str, list[Section]]]) -> list[epub.EpubHtml]:
+def compile_chapters(
+    chapters: list[tuple[str, list[Section]]]
+) -> Iterable[list[epub.EpubHtml]]:
+    chapter_digits = len(str(len(chapters)))
```

Comment:
Aren't we zero-indexing? Which is to say, with 10 chapters, the greatest id would be 9, not 10.

Reply:
I switched the chapter numbers to be 1-indexed as part of the general "increase filename human-readability" effort. I can switch them back if you'd rather stick with consistent zero-indexing, though. (And you're right that I made a mistake in this regard in the section_digits case, since that one _is_ meant to still be 0-indexed.)